src/App.js

```diff
@@ -0,0 +1,59 @@
+import './App.css';
+import {
+  Route, RouterProvider, createBrowserRouter, createRoutesFromElements, Outlet,
+} from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import Home from './pages/Home';
+import Session from './pages/Session';
+import AddCar from './pages/AddCar';
+import Navbar from './components/Navbar';
+import { fetchUser } from './redux/users/usersSlice';
+import Reservations from './pages/Reservations';
+import Reserve from './components/Reserve';
+import CarList from './components/CarList';
+import CarDetail from './components/CarDetail';
+import { getCars } from './redux/cars/carsSlice';
+
+const Root = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getCars());
+    const username = JSON.parse(localStorage.getItem('user_name'));
+    if (username) {
+      dispatch(fetchUser(username));
+    }
+  }, [dispatch]);
+  return (
+    <>
+      <Navbar />
+      <div className="container pt-5 mt-5 mt-lg-3">
+        <Outlet />
+      </div>
+    </>
+  );
+};
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<Root />}>
+      <Route index element={<Home />} />
+      <Route path="/session" element={<Session />} />
+      <Route path="/cars/new" element={<AddCar />} />
+      <Route path="/reservations/new" element={<Reserve />} />
+      <Route path="/reservations" element={<Reservations />} />
+      <Route path="/cars/delete" element={<CarList />} />
+      <Route path="/cars/:carId" element={<CarDetail />} />
+    </Route>,
+  ),
+);
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 4): 

![image](https://github.com/PabloEGonz/full-stack-capstone-front-end/assets/57421823/ff205c06-0273-42cd-a297-2d5fd2935357)

That way, your code will meet the latest standards. 💪